### PR TITLE
use the regular readme for the long description

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

**Comment by A5rocks** [comment]
Personally I like having a customized version of README for pypi even if it has caused issues due be being out of sync in the past.

(people seeing trio in pypi have different goals than people seeing trio on github. but idk if we adequately address that as is...)

**Comment by Fuyukai** [comment]
Genuinely, do people ever actually read the pypi readmes? I nearly always just click through to the repository instead (usually because the readme is outdated and wrong).

**Comment by A5rocks** [comment]
I'm not sure actually -- for the longest time the pypi readme wasn't rendered as rst and I don't remember anyone talking about it (I fixed it cause it annoyed me)

Maybe I'm just forgetting things though

File: pyproject.toml [hunk 1]
```diff
@@ -52,7 +52,7 @@ dependencies = [
 dynamic = ["version"]
 
 [project.readme]
-file = "LONG_DESCRIPTION.rst"
+file = "README.rst"
 content-type = "text/x-rst"
```
Comment (A5rocks) [inline pyproject.toml:56]: Do we really need to specify `content-type`?

**Review by A5rocks — APPROVED** [review]
OK I've come around on this, I agree with @Fuyukai that nobody really looks at these and that any upkeep burden is too much.

File: pyproject.toml (hunk 1 above)
Comment (CoolCat467) [reply pyproject.toml:56]: I think it's best practice, but I am not certain.

**Review by CoolCat467 — APPROVED** [review]
I think this makes sense

**Pull request merged**